composer.json

```diff
@@ -48,10 +54,6 @@
             "generateCookieValidationKey": [
                 "config/web.php"
             ]
-        },
-        "asset-installer-paths": {
```

Comment:
Of course that break it. And [you agree with that](https://github.com/yiisoft/yii2-app-advanced/commit/785c933ba31c0827dfdab92363898b461d7d40e3#commitcomment-21724551)!

> @WadeShuler : It looks like the master branch, and next release of Yii2, will need:
composer global require "fxp/composer-asset-plugin:^1.3.0"
> @samdark : Yes. We can update docs right now, I think, to have ~1.3.1.

And today we have two official templates (basic and advanced) with different backward compatibility. 🤷‍♂️